openqaoa/devices.py

```diff
@@ -297,13 +299,20 @@ def __init__(self, device_name: str, folder_name: str = 'openqaoa'):
         ----------
 		device_name: `str`
 			The ARN string of the braket QPU/simulator to be used
+        s3_bucket_name: `str`
+            The name of S3 Bucket where the Braket run results will be saved.
```

Comment:
Is this default also selected by what's set up by the CLI?

Reply:
You cant set up the bucket name from the CLI AFAIK. 
The default naming for the bucket is created by the sdk we are using, so the bucket name would be something like `amazon-braket-REGION-SSID`